python/cross_service/photo_analyzer/README.md

```diff
@@ -0,0 +1,169 @@
+#  Analyzing photos using Amazon Rekognition with the AWS SDK for Python
+
+## Overview
+
+Shows you how to use the AWS SDK for Python (Boto3) to create a web application that 
+lets you upload photos to an Amazon Simple Storage Service (Amazon S3) bucket, use 
+Amazon Rekognition to analyze and label the photos, and use Amazon Simple Email 
+Service (Amazon SES) to send email reports of image analysis.  
+
+This example contains two main components: a web page written in JavaScript that is built
+with React, and a REST service written in Python that is built with Flask-RESTful.
+
+You can use the React web page to:
```

Comment:
You can use the React webpage to do the following:

Reply:
done